core/types/transaction_signing.go

```diff
@@ -170,6 +175,75 @@ type Signer interface {
 	Equal(Signer) bool
 }
 
+type cancunSigner struct{ londonSigner }
```

Comment:
```suggestion
type londonSignerWith4844 struct{ londonSigner }
```

Let's do this instead of `cancunSigner` and change occurrences of `NewLondonSigner` to `NewLondonSignerWith4844`.

Reply:
makes sense. changed in [42452b4](https://github.com/scroll-tech/go-ethereum/pull/626/commits/42452b4b1e7c67295418aba71dee688ac782fde0)